csaf_files/IT/white/2025/va-25-120-01.json

```
@@ -0,0 +1,309 @@
+{
+    "document": {
+        "category": "csaf_vex",
+        "csaf_version": "2.0",
+        "lang": "en-US",
+        "publisher": {
+            "category": "coordinator",
+            "contact_details": "https://www.cisa.gov/report",
+            "issuing_authority": "CISA",
+            "name": "CISA",
+            "namespace": "https://www.cisa.gov/"
+        },
+        "title": "Commvault Web Server unspecified vulnerability",
+        "tracking": {
+            "current_release_date": "2025-05-02T02:22:31Z",
+            "generator": {
+                "engine": {
+                    "name": "VINCE-NT",
+                    "version": "1.8.0"
+                }
+            },
+            "id": "VA-25-120-01",
+            "initial_release_date": "2025-05-02T02:22:31Z",
+            "status": "draft",
```

@amanion-cisa should be changed to "final"

```suggestion
            "status": "final",
```

---

FIxed in 02a85f85c38f27feb180cd954a0ef922ff62de7f.